app/components/conferences/admin/SponsorForm.js

```diff
@@ -0,0 +1,365 @@
+                                        name="description"
+                                        as="textarea"
+                                        type="textarea"
+                                        placeholder="A short sweet biography"
+                                        defaultValue={spon.attributes.description}
+                                    />
+                                </Form.Group>
+                                <Form.Group className="mb-3">
+                                    <Form.Label>URL</Form.Label>
+                                    <Form.Control
+                                        type="url"
+                                        onChange={e => {props.handleChange(e,id)}}
+                                        name="url"
+                                        placeholder="https://link-to.image"
+                                        defaultValue={spon.attributes.url}
+                                    />
+                                </Form.Group>
+                                <Form.Group className="mb-3">
+                                    <Form.Label>Logo-Url</Form.Label>
+                                    <Form.Control
+                                        type="url"
+                                        onChange={e => {props.handleChange(e,id)}}
+                                        name="logo-url"
+                                        placeholder="https://link-to.image"
+                                        defaultValue={spon.attributes["logo-url"]}
+                                    />
+                                </Form.Group>
+                                <InputGroup className="mb-3">
+                                    <InputGroup.Text>Level</InputGroup.Text>
+                                    <Form.Control
+                                        type="number"
+                                        onChange={e => {props.handleChange(e,id)}}
+                                        name="level"
+                                        placeholder="0"
+                                        defaultValue={spon.attributes.level}
+                                    />
+                                    <InputGroup.Text>Type</InputGroup.Text>
+                                    <Form.Control
+                                        type="text"
+                                        onChange={e => {props.handleChange(e,id)}}
+                                        name="type"
+                                        placeholder="Gold"
+                                        defaultValue={spon.attributes.type}
+                                    />
+                                </InputGroup>
+                            </Container>
+                        </ListGroupItem>
+                        </div>
+                        );
+                    })
+                    : "No Sponsor found"}
```

Comment:
This text should be something like greyed out or italic. To differentiate with other texts. 

Or I would highly suggest having a toggle switch with title "Enable Sponsors," which by default will be toggled off, but when toggled on, it should show the question forms embedded inside the component, not as modal.

Thank you!

Reply:
I have changed the text in italic and also changed the color to grey.